packages/tables/src/elements/Table.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledTable, IStyledTableProps } from '../styled';
+
+/**
+ * Accepts all `<table>` attributes and events
+ */
+export const Table = React.forwardRef<
```

Comment:
I noticed that there are [existing](https://github.com/zendeskgarden/react-components/blob/next/packages/chrome/src/elements/Chrome.spec.tsx#L26) [unit](https://github.com/zendeskgarden/react-components/blob/next/packages/grid/src/elements/Grid.spec.tsx#L19) [tests](https://github.com/zendeskgarden/react-components/blob/next/packages/tabs/src/elements/Tab.spec.tsx#L18) which assert that components exported to consumers should forward refs.

Should this PR include unit test for `Table` ( and `Body`, `Caption`, `Cell`, `GroupRow`, `Head`, `HeaderCell`, `HeaderRow`, `OverflowButton`, `Row`) now that these components have been updated to support ref forwarding?

Reply:
I've now added these `ref` checks. I also found a invalid DOM nesting warnings from `react-dom` (thanks react-testing-library 🎉 ). I've updated the `Cell/Row` examples to include wrapping `Table/Body` elements